File: Projeto_1/teste.js
import Cliente from "./cliente.js";

// criando uma nova instancia da classe Cliente
const cliente = new Cliente("João Silva", 30, 1000.0);

//mostrando as informacoes
cliente.mostrarInformacoes();

//atualizando o saldo
cliente.atualizarSaldo(500); 
cliente.atualizarSaldo(-200); 

//mostrando de novo depois de atualizar
cliente.mostrarInformacoes();

//alterando usando os setters para acessar os atributos (no js esse _ pra deixar o atributo privado eh mais uma convencao, por isso, usar setters e getters)
cliente.nome = "Maria Oliveira";
cliente.idade = 28;
cliente.saldo = 2000.0;

//mostrando de novo depois de alterar
cliente.mostrarInformacoes();


// COMO RODAR: 
//PRECISA TER O NODEJS INSTALADO
// NO TERMINAL DENTRO DA PASTA PROJETO_1 DIGITE: node teste.js
// ~/poo-bvg/Projeto_1$ node teste.js

File: Projeto_1/teste.py
import classes as c

Cliente1 = c.cliente("Marcos", 37, 5800)
Cliente2 = c.cliente("Juliana", 24, 10200)

#mostrar as informacoes do marcos e da julia
Cliente1.mostrar_informacoes() 
Cliente2.mostrar_informacoes()

#atualizar o saldo dos dois clientes
Cliente1.atualizar_saldo(2300) #saldo de 8100
Cliente2.atualizar_saldo(680) #saldo de 10880